scripts/install-chef-suse.sh

```diff
@@ -290,6 +312,10 @@ then please remove the following file:
     /opt/dell/crowbar_framework/.crowbar-installed-ok
 EOF
     exit 1
+elif [ -f /opt/dell/crowbar_framework/.crowbar-install-failed ] || [ "$CROWBAR_FIRST_INSTALL" ]; then
+    rm -f /opt/dell/crowbar_framework/.crowbar-install-failed
+    cleanup_steps
+    sqlite3 /opt/dell/crowbar_framework/db/production.sqlite3 "delete from proposals; delete from proposal_queues"
```

Comment:
Also, can't we have a better way of cleaning the database? Right now, every time we add a table, we'll have to update this, which is painful...